src/coreComponents/physicsSolvers/multiphysics/CoupledSolver.hpp

```diff
@@ -42,6 +57,17 @@ class CoupledSolver : public SolverBase
                  Group * const parent )
     : SolverBase( name, parent )
   {
+    /// GEOS mainly uses FIM coupling so let's define FIM as the default.
+    registerWrapper( viewKeyStruct::couplingTypeString(), &m_couplingType ).
+      setInputFlag( dataRepository::InputFlags::OPTIONAL ).
+      setApplyDefaultValue( CouplingType::FIM ).
+      setDescription( "Type of coupling. Options are: Sequential and FIM" );
+
+    registerWrapper( viewKeyStruct::subcyclingOptionString(), &m_subcyclingOption ).
+      setInputFlag( dataRepository::InputFlags::OPTIONAL ).
+      setApplyDefaultValue( 0 ).
```

Comment:
Maybe we should put the default option to `subcyclingOption = 1` because without subcycling, the results may be not converged and potentially inaccurate?

Reply:
The issue is that the `subcycling` in not very robust for now because if one of the solvers cuts the timestep things may go wrong. For now, the main sequential solver is phaseField for which we don't usually use subcycling. I think we can change it when we merge the sequential thermoporomechanics.